test/acceptance/txIdFromHeader.test.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: loopback-next-hello-extension
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Application, get} from '@loopback/core';
+import {txIdFromHeader} from '../..';
+import {Client, createClientForApp} from '@loopback/testlab';
+
+describe('@txIdFromHeader() tests', () => {
```

Comment:
Please move this file to `test/acceptance/decorators/txIdFromHeader.acceptance.ts`. 
 - We should use the same directory structure as in `src` for decorators/controllers/providers etc. 
 - The test file name should use a different type suffix for acceptance and unit tests. In loopback-next core, we are using `.acceptance.ts`, `.integration.ts` and `.test.ts`, I think we should do the same here. Maybe use `.unit.ts` instead of `.test.ts`?

For the top-level `describe` block, our convention in loopback-next is to add "(acceptance)" or "(unit)" to the name, e.g. `describe('@tsIdFromHeader() (acceptance)')`.